hazelcast/src/test/java/com/hazelcast/client/test/IdentifiedFactory.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ * Copyright (c) 2008, Hazelcast, Inc. All Rights Reserved.
```

Comment:
The copyright should include the year range, so `Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.`